resources/js/pages/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,11 @@
+import { Fragment } from "react";
+
+import Navbar from "./components/Navbar";
+
+export default function Dashboard() {
```

Comment:
if directory is under the same name `dashboard` please call the file `index.js`

Reply:
Changed the main folder files to `index.tsx`